Extensions/PlayerAuthentication/playerauthenticationcomponents.ts

```diff
@@ -1,12 +1,39 @@
 namespace gdjs {
   const logger = new gdjs.Logger('Player Authentication');
   export namespace playerAuthenticationComponents {
+    const getMessages = ({
+      platform,
+      registered,
+    }: {
+      platform: 'cordova' | 'electron' | 'web';
+      registered: boolean;
```

Comment:
`isGameRegistered`? One could you think you're talking about the register state of the player